packages/dev_disc/README.md

```diff
@@ -107,6 +105,8 @@ retry = 5
 keep_alive = 60
 
 ConnectDisconnectTimeout = 10
+* When MQTT broker also sits on the edge system, MQTT subscriber can listen on 127.0.0.1/localhost with port < 1024 and use basic authentication
```

Comment:
security team asked me to use port < 1024 to be securer. However, I tested many times on my mac. If we set listen port < 1024 for MQTT broker, the connection from any client (a test tool) will be rejected.
Of course, our liota dev_comms connection will also be rejected.
I'll modify that back, while mention in README.md that users had better set a special port # rather than well-known 1883.